client/src/components/Navbar/NavDropDown.jsx

```diff
@@ -26,24 +27,30 @@ const NavDropDown = () => {
     };
   }, [clickRef]);
 
-  const toggleDropDown = e => {
+  const toggleDropDown = (e) => {
     e.preventDefault();
     e.stopPropagation();
-    setLoggedInMenuHidden(state => !state);
+    setLoggedInMenuHidden((state) => !state);
   };
 
-  const handleLogout = e => {
+  const handleLogout = (e) => {
     e.preventDefault();
     dispatch(logOut());
   };
   return (
     <div className="flex flex-col relative" ref={clickRef}>
-      <img
-        src={user.profilePic || avatar}
-        alt=""
-        className="h-14 w-14 cursor-pointer border-2 border-baby rounded-full object-cover"
-        onClick={e => toggleDropDown(e)}
-      />
+      <div
+        className="flex items-center cursor-pointer"
+        onClick={(e) => toggleDropDown(e)}
+      >
+        <img
+          src={user.profilePic || avatar}
+          alt=""
+          className="h-14 w-14 border-2 border-baby rounded-full object-cover"
+        />
+        <IoMdArrowDropdown className="text-xl" />
```

Comment:
I don't think the Arrow is necessary and kind of takes away from the minimal nature of the design, it looks like it's there by accident as it's not an accordion menu

Reply:
I was on the profile page and I didn't know how to navigate to any other page. I was confused until I hovered on the avatar and I saw the pointer change.

This is me. One of the people that worked on this. Now imagine a user navigating through the page. I added the dropdown to show that it is a menu and it would draw to hover or click on it. The image the way it was at first does not look like it has any other functionality. That's what I think

Reply:
I think it's quite a common interface for menu. Maybe we should take a pic of both options and post them in the discord for a team vote? Otherwise could just use the burger menu, maybe?

Reply:
Alright. That works. We can have the other guys decide on which they prefer